.github/workflows/publish-sample-add-ons.yml

```diff
@@ -0,0 +1,70 @@
+name: Deploy sample Add-ons to GitHub Pages
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["main"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [22.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+    steps:
+    - uses: actions/checkout@v4
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+        cache-dependency-path: addons-web-sdk/samples/hello-world/package-lock.json
+    # TODO: run this in a loop when there are multiple samples.
```

Comment:
I might suggest using https://docs.npmjs.com/cli/v8/using-npm/workspaces instead of looping here

Reply:
I'm not actually sure if this is a good usecase of workspaces, but bears investigating. Normally I've used them when working on multiple packages in one larger project, but these are mostly-unrelated samples, so YMMV.

Reply:
Ooh, thanks for sharing that. Good idea. I think that would work for how simple this NPM config is. 

I just updated the TODO for now, because I will want to do more involved testing of the GH action a la https://github.com/googleworkspace/meet/pull/12 once this approach of using GH pages is approved/merged